src/app/Components/Artist/Articles/Articles.tsx

```diff
@@ -19,7 +19,13 @@ export const Articles: React.FC<Props> = ({ articles, artist }) => {
   return (
     <Flex>
       <Flex flexDirection="row" justifyContent="space-between">
-        <Text variant="sm-display">{`Artsy Editorial Featuring ${artistData.name}`}</Text>
+        <Flex maxWidth="80%">
```

Comment:
I believe it's better to use `flex={1}` here. The remaining 20% of the screen might not provide enough space for the "View All" button on small devices.

```suggestion
        <Flex flex={1} pr={1}>>
```

Reply:
makes sense

Reply:
done

Reply:
What do you think about adding `pr={1}` as well to get some space between the `...` and the "View All"?